ttnn/cpp/ttnn/operations/unary.hpp

```diff
@@ -95,16 +94,13 @@ struct Softplus : public EltwiseUnary {
         return detail::input_tensors_to_validate(input_tensor, std::forward<Args>(args)...);
     }
 
-    static Tensor execute(const Tensor& input, const float beta, const float threshold, const std::optional<MemoryConfig>& memory_config_arg = std::nullopt) {
-        auto original_input_shape = input.get_shape();
-        auto input_4D = ttnn::unsqueeze_to_4D(input);
```

Comment:
Is unsqueeze and squeeze(reshape) no longer needed with this update? Or were they not necessary prior to this?

Reply:
Unsure about previously, but it is now not necessary because we are just invoking a single EltwiseUnary call.

Reply:
@ayerofieiev-tt can you also take a look here? You'd recently ported over from python.

Reply:
Previously, one of the ops, from which softplus was combined, called for padding and it required 4D tensor.
I am sure this is now not required, which is awesome!